concreate/generator.py

```diff
@@ -117,15 +156,17 @@ def build(self):
             2. 'latest'
         """
         # Desired tag of the image
-        tag = "%s:%s" % (self.effective_descriptor['name'], self.effective_descriptor['version'])
+        tag = "%s:%s" % (self.effective_descriptor[
+                         'name'], self.effective_descriptor['version'])
         latest_tag = "%s:latest" % self.effective_descriptor['name']
 
         logger.info("Building %s container image..." % tag)
 
-        ret = subprocess.call(["docker", "build", "-t", tag, "-t", latest_tag, os.path.join(self.target, 'image')])
+        ret = subprocess.call(["docker", "build", "-t", tag,
```

Comment:
why not check call if are using ret as rc later?

Reply:
OK